testing/xfce4-sensors-plugin/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Maintainer:
```

Comment:
I found recent examples of packages being added to /testing without a maintainer.
https://github.com/alpinelinux/aports/commit/f4a05b94a60187d1346956c1c284539ea1d73ad2
https://github.com/alpinelinux/aports/commit/1ce94bc0259bc8026301433dd51c1970fb4a1810
https://github.com/alpinelinux/aports/commit/b375dc18dc8bccafe490bf5c58d80543cbb53910
https://github.com/alpinelinux/aports/commit/3d60f5daf2c89dd871b57ac292ec69aa955ef885